packages/web/src/components/layouts/SearchResults/SearchResults.tsx

```diff
@@ -92,6 +115,21 @@ export default function SearchResults({
     },
   ];
 
+  const handleSortChange = useCallback(
+    (newSorterName: PackageSortType) => {
+      const sortOrder = newSorterName === sortField && sortDirection === 'DESC' ? 'ASC' : 'DESC';
```

Comment:
If user clicks on sort button that is already active (by severity, for example) it will change direction to a opposite. If user clicks on a sort that is not active, it will active it and set direction to `DESC`

If you would left just the `sortDirection === 'DESC' ? 'ASC' : 'DESC'` part clicking on a new sort method, it will change sort method and order simultaneously, ending up in confusing UI like that:
![19ef2d8d35](https://user-images.githubusercontent.com/2999208/196410222-080297c6-342a-42e1-a9ab-c55ce0c79ba4.gif)